skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.kt

```diff
@@ -20,11 +22,47 @@ private class PictureHolder(val instance: Picture, val width: Int, val height: I
 
 open class SkiaLayer(
     private val properties: SkiaLayerProperties = SkiaLayerProperties()
-) : HardwareLayer() {
+) : JPanel() {
+
+    val backedLayer : HardwareLayer
+
+    init {
+        setOpaque(false)
```

Comment:
Why cannot we use setter?

Reply:
JComponent has "setOpaque" and "isOpaque" methods, so we cannot use "opaque" as setter/getter.